test_data/GetMetrics_group_by_queryid.json

```diff
@@ -12,7 +12,7 @@
 			"sum": 75710,
 			"min": 113,
 			"max": 113,
-			"avg": 1130,
```

Comment:
interesting case, how avg can be different then min max if they are equal?

Reply:
Let me explain:

If log_slow_rate_limit > 1 - MySQL randomly log every 1/log_slow_rate_limit query.
So if log_slow_rate_limit = 100, only 1% of a random query with specific queryid is logged in slow log.

For example, in one minute we catch bytes_sent 5 times with values [10, 10, 10, 10, 10].
In go-mysql we calculate it in the next way:

bytes_sent_cnt = 5  
num_queries = count * log_slow_rate_limit  = 5 * 100 = 500

bytes_sent_sum =  (10 + 10 + 10 + 10 + 10) * 100 = 5000
bytes_sent_min = 10 - this is a guess min of what we catch. real min can be 0.
bytes_sent_max = 10 - this is a guess max of what we catch. real min can be 1000.

In qan-api we need to calculate it as:
bytes_sent_avg = bytes_sent_sum / num_queries = 5000 / 500 =  10

For current velue:
num_queries = 340703
bytes_sent_sum = 75710
bytes_sent_avg = bytes_sent_sum / num_queries = 75710 / 340703 = 0.2222170043

https://github.com/percona/qan-api2/blob/38f3d6af2f96781392afe4a237b4cefba82ed724/test_data/GetMetrics_group_by_queryid.json#L80